### ISSUE TITLE
discuss Aphelele rates

### ISSUE THREAD

**Issue by CIEspost** [zacck/cieproductions#35]
@ApheleleM has agreed to work on the things while getting taught the 360 skills in return right now and we all @zacck as our witness have agreed that the moment we start making money we will discuss the rates with Aphelele. @ApheleleM is this statement good with you?

**Comment by CIEspost** [comment]
Aphelele mail: 

[Aphelele Mantshiyose - Invoice 360 video (1).pdf](https://github.com/zacck/cieproductions/files/1153070/Aphelele.Mantshiyose.-.Invoice.360.video.1.pdf)
I hope this finds you well.

Please find the attached Invoice, 

We both know I don't know how long it take to stitch and editing 360 video, it was hard for me to do an invoice from something I don't even know how it works.

Hence you guys really needed it then, I had to deliver but if you see a need of change please don't hesitate to inform me.

I believe once I start editing I will know how to invoice you fair.

**Comment by ApheleleM** [comment]
@CIEspost Yes we will do so, and I have the Laptop with me

**Comment by CIEspost** [comment]
@ApheleleM awesome i will meet you there by the stairs at 5:15 then thanks

**Comment by CIEspost** [comment]
@ApheleleM i think it is important for us to keep track of this, as this will always be important even if you have shares, it is important for us as shareholders to know when we budget for projects how much we want to ask for our individual skills

**Comment by ApheleleM** [comment]
@CIEspost  Yes I agree with you, we need to come up with a mutual understand of how to go about it.
